code/controllers/master.dm

```diff
@@ -615,24 +657,23 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 
 
 /datum/controller/master/stat_entry(msg)
-	msg = "(TickRate:[Master.processing]) (Iteration:[Master.iteration]) (TickLimit: [round(Master.current_ticklimit, 0.1)])"
+	msg = "(TickRate:[Master.processing]) (Iteration:[Master.iteration]) (TickLimit: [round(Master.current_ticklimit, 0.1)]) (Average Starting Tick Usage: [round(Master.average_starting_tick_usage, 0.1)] \
+	(Average Sleeping TIck Usage: [round(Master.average_sleeping_tick_usage, 0.1)]) (Average Sleeping Overtime: [round(Master.average_sleeping_overtime_usage)]))"
```

Comment:
Do we need the average here? more similar words there are the harder it is to read at a glance

Reply:
i shortened it to Avg ill see if that makes it better as i adjust it more (and add more stats)